scripts/environment-unpinned.yaml

```
@@ -29,3 +29,4 @@ dependencies:
       - jsonschema
       - plyvel
       - domain-utils
+      - dataclasses-json==0.5.2
```

Please don't pin the dependency here unless you have a strong reason to do so.

---

Hmm... I am not sure if I understand what pinning this dependency does here... 
How else would I add new dependency to environment.yaml?

---

```suggestion
      - dataclasses-json
```
Without naming a specific version. This way `repin.sh` will just pick out the latest version whenever it is run.